autoload/kite/onboarding.vim

```diff
@@ -0,0 +1,81 @@
+let s:text = [
+      \ 'Kite is now integrated with Vim',
+      \ '',
+      \ 'Kite is an AI-powered programming assistant',
+      \ 'that shows you the right information at the right',
+      \ 'time to keep you in the flow.',
+      \ '',
+      \ 'Please choose:',
+      \ '',
+      \ 'Learn how to use Kite',
+      \ 'Hide',
+      \ 'Hide forever',
+      \ ]
+let s:option = 'onboarding_required'
+
+
+function! kite#onboarding#call()
+  if !kite#utils#get_setting(s:option, 1)
+    return
+  endif
+
+  call kite#client#onboarding_file(function('kite#onboarding#handler'))
+endfunction
+
+
+function! kite#onboarding#handler(response) abort
+  if a:response.status == 200
+    silent execute 'tabedit' s:trim(a:response.body, '"')
```

Comment:
What is trimmed here?

Reply:
When I implemented this the response body was `"/path/to/some/file"` instead of `/path/to/some/file` so I had to get rid of the enclosing double-quotation marks.

Reply:
I see. As far as I know this is JSON, so it could be possible that this path contains escaped characters. I'm not sure if this will happen in practice, though. 
If it's a quick change, could you parse this string as JSON and use the result as path?

Reply:
Ah, I didn't realise it was JSON.  I've updated the code (9162342).